pkgs/by-name/sq/sqlitestudio-plugins/package.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  python3,
+
+  sqlitestudio,
+}:
+stdenv.mkDerivation {
+  pname = "sqlitestudio-plugins";
+
+  inherit (sqlitestudio)
+    version
+    src
+    nativeBuildInputs
+    buildInputs
+    ;
+
+  postConfigure = ''
+    uic ./SQLiteStudio3/guiSQLiteStudio/mainwindow.ui -o ./SQLiteStudio3/guiSQLiteStudio/ui_mainwindow.h
+  '';
+
+  qmakeFlags = [
+    "./Plugins"
+    "PYTHON_VERSION=${python3.pythonVersion}"
+    "INCLUDEPATH+=${python3}/include/python${python3.pythonVersion}"
+  ];
+
+  meta = {
+    description = "Official plugins for SQLiteStudio, a free, open source, multi-platform SQLite database manager";
+    homepage = "https://sqlitestudio.pl/";
+    license = lib.licenses.gpl3;
+    platforms = lib.platforms.linux;
+    badPlatforms = [ lib.systems.inspect.platformPatterns.isStatic ];
+    maintainers = with lib.maintainers; [ asterismono ];
+  };
```

You can

```suggestion
  meta = sqlitestudio.meta // {
    description = "Official plugins for SQLiteStudio, a free, open source, multi-platform SQLite database manager";
    badPlatforms = [ lib.systems.inspect.platformPatterns.isStatic ];
  };
```


While the license should be `gpl3Only` as the author doesn't state "or any later versions" but explicitly mentioned the version elsewhere. This can be changed in `sqlitestudio`.

---

Should this be `gpl3Plus` instead?

```
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.
```
 [SQLiteStudio LICENSE](https://github.com/pawelsalawa/sqlitestudio?tab=License-1-ov-file#:~:text=This%20program%20is%20free%20software%3A%20you%20can%20redistribute%20it%20and/or%20modify%0A%20%20%20%20it%20under%20the%20terms%20of%20the%20GNU%20General%20Public%20License%20as%20published%20by%0A%20%20%20%20the%20Free%20Software%20Foundation%2C%20either%20version%203%20of%20the%20License%2C%20or%0A%20%20%20%20(at%20your%20option)%20any%20later%20version.)

[Ref](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206)

Or should I contact the upstream package author first, as stated in the guideline `the packager should always ask the author.` found [here](https://github.com/NixOS/nixpkgs/issues/304000)?

---

Thank you! I missed that — I’ll correct the license field.